lightning/src/chain/onchaintx.rs

```diff
@@ -162,6 +164,19 @@ impl Writeable for Option<Vec<Option<(usize, Signature)>>> {
 	}
 }
 
+#[derive(PartialEq, Eq, PartialOrd, Ord)]
+pub(crate) enum ClaimEvent {
+	BumpCommitment {
+		target_feerate_sat_per_1000_weight: u32,
+		commitment_tx: Transaction,
+		anchor_output_idx: u32,
```

Comment:
The event could return the expiration timelocks and the value of all HTLCs if the user would like to be more aggressive in term of fee-bumping than our default suggestion? Or maybe we could expose some internal setting like a fee-bumping frequency `low,medium,high`. Ultimately it's a choice between fee-savings and risk of non-confirmation.